code/game/atoms.dm

```diff
@@ -2103,15 +2103,53 @@
 	SHOULD_NOT_SLEEP(TRUE)
 
 	var/mob/user = client?.mob
+	if (isnull(user))
+		return
 
 	// Screentips
-	var/datum/hud/active_hud = user?.hud_used
+	var/datum/hud/active_hud = user.hud_used
 	if(active_hud)
```

Comment:
Should just early return here

Reply:
The reason I didn't is because this proc can theoretically hold a lot, I reserve early-returns for when no code will ever go past that point.